addons/mail/static/tests/qunit_suite_tests/components/attachment_box_tests.js

```diff
@@ -27,16 +17,6 @@ QUnit.test('base empty rendering', async function (assert) {
         threadId: resPartnerId1,
         threadModel: 'res.partner',
     });
-    assert.strictEqual(
-        document.querySelectorAll(`.o_AttachmentBox`).length,
-        1,
-        "should have an attachment box"
-    );
-    assert.strictEqual(
-        document.querySelectorAll(`.o_AttachmentBox_buttonAdd`).length,
-        1,
-        "should have a button add"
-    );
```

Comment:
This looks wrong, when `isAttachmentBoxVisibleInitially` is true it would be expected that there is an attachment box.
This is related to the "bad" condition added in the template, instead of better controlling the presence of the attachment box record.